en/docs/get-started/quick-start-guide/integration-qsg.md

```diff
@@ -206,22 +235,49 @@ Upon invocation, you should be able to observe the following response:
 ]
 ```
 
-## Step 3 - Discover services
+## Step 3 - Exposing an Integration Service as a Managed API
 
-When you deploy your C-App it becomes an integration service and you can view your service in a Service Catalog. 
+After developing your Integration using WSO2 Micro Integrator, you can choose to expose it as a Managed API using WSO2 API Manager.
 
 Integration services are made discoverable to the API Management layer via the Service Catalog so that API proxies can directly be created using them.
 
+For this part of the tutorial you need WSO2 API Manager.
+
+1.  Download and set up [WSO2 API Manager 4.0.0](https://wso2.com/api-management/).
+2.  Start WSO2 API Manager by navigating to the /bin directory using the command-line and execute the following command `api-manager.bat --run` (for Windows) or `sh api-manager.sh` (for Linux/Mac)
+
+You need to add the following configuration to `<MI_HOME>/conf/deployment.toml` file.
+
+!!! Tip
+    The default username and password for connecting to the API gateway is `admin`.
+
+
+```toml
+[[service_catalog]]
+apim_host = "https://localhost:9443"
+enable = true
+username = "admin"
+password = "admin"
+```
+    
+Now if you restart Micro Integrator, you can observe that API will be added to the Service Catalog of API Manager.
+You will see the following in the server start up log.
+
+```bash
+Successfully updated the service catalog
```

Comment:
Shouldn't this message change if we are moving away from the term service catalog.

Reply:
This message is already there in the Product. Hence this cannot be changed at the moment